spec/_attachments/ic.did

```diff
@@ -7,6 +7,7 @@ type canister_settings = record {
     memory_allocation : opt nat;
     freezing_threshold : opt nat;
     reserved_cycles_limit : opt nat;
+    wasm_heap_memory_limit : nat;
```

Comment:
Let's go with `wasm_memory_limit` which is shorted and would allow us to add `stable_memory_limit` in the future. (Sorry if it was not clear from the ticket).